platforms/documentation/docs/src/snippets/dependencyManagement/workingWithDependencies-iterateArtifacts/kotlin/build.gradle.kts

```diff
@@ -14,9 +14,11 @@ tasks.register("iterateResolvedArtifacts") {
     val scm = configurations["scm"]
     dependsOn(scm)
 
+    val scmArtifacts = scm.map { it.absolutePath }
+    val logger = logger
```

Comment:
💅 The logger does not seem to be required to be moved out of the doLast in the Groovy sample. Is it required to be moved out in Kotlin?